lib/smart_proxy_dynflow/continuous_output.rb

```diff
@@ -37,18 +37,20 @@ def humanize
       raw_outputs.map { |output| output['output'] }.join("\n")
     end
 
-    def add_exception(context, exception, timestamp = Time.now.getlocal)
-      add_output(context + ": #{exception.class} - #{exception.message}", 'debug', timestamp)
+    def add_exception(context, exception, timestamp: Time.now.getlocal, id: nil)
+      add_output(context + ": #{exception.class} - #{exception.message}", 'debug', timestamp: timestamp, id: id)
     end
 
-    def add_output(*args)
-      add_raw_output(self.class.format_output(*args))
+    def add_output(*args, **kwargs)
+      add_raw_output(self.class.format_output(*args, **kwargs))
```

Comment:
Would it make sense to also just forward the arguments here?
```suggestion
    def add_output(...)
      add_raw_output(self.class.format_output(...))
```